setup.py

```diff
@@ -5,7 +5,7 @@
 import sys
 
 extra_install_requires = []
-extra_tests_require = []
+extra_tests_require = ['httpretty']
```

Comment:
Instead of having this here, put it inside `tests_require`
Below `pytz`

These lists are intentionally empty then appended to based on if a specific package is needed for a specific version of python.

I should probably spell this out more with documentation.